static/app/components/events/interfaces/crashContent/exception/rawContent.tsx

```diff
@@ -171,7 +171,7 @@ class RawContent extends React.Component<Props, State> {
             return null;
           }
           return (
-            <div key={excIdx}>
+            <div key={excIdx} aria-label={t('Raw stack trace')}>
```

Comment:
What does a screen reader do with aria-label on div?

Reply:
you're right.. `aria-label` should only be placed on interactive elements or elements with interactive roles... I'll replace it with `data-test-id`